cmd/show/contributors.go

```diff
@@ -0,0 +1,156 @@
+package show
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	client "github.com/open-sauced/go-api/client"
+)
+
+// ContributorModel holds all the information related to a contributor
+type ContributorModel struct {
+	userInfo *client.DbUser
+	userPrs  []client.DbPullRequest
+}
+
+// BackMsg: message to signal main model that we are back to dashboard when backspace is pressed
+type BackMsg struct{}
+
+// InitContributor: initializes the contributorModel
+func InitContributor(contributorName string) (ContributorModel, tea.Cmd) {
+	var contributorModel ContributorModel
+	var wg sync.WaitGroup
+
+	wg.Add(2)
+	go func() {
+		defer wg.Done()
+		// TODO: handle error
+		userInfo, _ := fetchContributorInfo(contributorName)
+		contributorModel.userInfo = userInfo
+
+	}()
+
+	go func() {
+		defer wg.Done()
+		// TODO: handle error
+		userPRs, _ := fetchContributorPRs(contributorName)
+		contributorModel.userPrs = userPRs
+	}()
+
+	wg.Wait()
+
+	return contributorModel, func() tea.Msg { return SuccessMsg{} }
+}
+
+func (m ContributorModel) Init() tea.Cmd {
+	return nil
+}
+
+func (m ContributorModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	var cmd tea.Cmd
+	switch msg := msg.(type) {
+	case tea.KeyMsg:
+		switch msg.String() {
+		case "backspace":
+			return m, func() tea.Msg { return BackMsg{} }
+		case "q", "esc":
+			return m, tea.Quit
+		}
+	}
+	return m, cmd
+}
+
+func (m ContributorModel) View() string {
+	if m.userInfo != nil {
+		return m.drawContributorView()
+	}
+	return "🚧  Not found"
```

Comment:
This was all I got when hitting enter on some alumni contributors and then it seemed to freeze and not take anymore input. Maybe this is a non-starter return or blocks somehow?

Reply:
Interesting, usually when this hits 'Not found' you can go back to the dashboard model with 'backspace' maybe adding the help menu could be useful for navigation ? I've seen this pattern in other bubbletea programs. I am restructuring this to avoid this nil checking on the contributor and only render valid contributors by and giving a quick response via the dashboard model like this.

<img width="1219" alt="Screenshot 2023-09-01 at 1 30 18 PM" src="https://github.com/open-sauced/pizza-cli/assets/59541661/e949232a-417a-4f11-8f50-df5070289496">